internal/p2p/options.go

```diff
@@ -224,29 +223,23 @@ func Discovery(bootstrapAddrs []multiaddr.Multiaddr) Options {
 
 func Monitor() Options {
 	return func(n *Node) error {
-		output := func() {
-			fmt.Printf(
-				`{"id":"%s","key":"connectedPeerCount","val":%d,"ts":%d}`+"\n",
-				n.host.ID().String(),
-				len(n.host.Network().Peers()),
-				time.Now().Unix(),
-			)
-		}
-		ticker := func() {
-			t := time.NewTicker(time.Minute)
-			for {
-				output()
-				select {
-				case <-n.ctx.Done():
-					t.Stop()
-					return
-				case <-t.C:
-				}
-			}
-		}
 		n.AddNodeEventHandler(sets.NodeEventHandlerFunc(func(event interface{}) {
 			if _, ok := event.(sets.NodeStartedEvent); ok {
-				go ticker()
+				go func() {
+					t := time.NewTicker(time.Minute)
+					for {
+						n.log.
+							WithField("hostId", n.host.ID().String()).
```

Comment:
Printing a current peer ID every minute looks a little useless

Reply:
The idea is to not have to edit the logs (add id) when I push it to monitoring).
This way I have no dependencies on the instance of spire.

Reply:
It looks like a little hacky solution for a problem to me ;/ Maybe it helps if we add the `spire id` command?

Reply:
> Maybe it helps if we add the spire id command?

That is exactly what I want to avoid.